maraboupy/test/test_query.py

```diff
@@ -100,6 +102,28 @@ def test_to_query(tmpdir):
     assert stats_net.hasTimedOut()
     assert stats_ipq.hasTimedOut()
 
+
+def test_get_marabou_query(tmpdir):
+    '''
+    Tests that input query generated from a network in Maraboupy is identical to the input query generated directly by
+    Marabou from the same network file (relies on saveQuery, compares the output of saveQuery on the two queries).
+    '''
+    network = load_acas_network()
+    ipq1 = network.getMarabouQuery()
+
+    ipq2 = MarabouCore.InputQuery()
+    network_filename = os.path.join(os.path.dirname(__file__), ACAS_FILE)
+    MarabouCore.createInputQuery(ipq2, network_filename, '')
+
+    ipq1_filename = tmpdir.mkdir("query").join("query1.txt").strpath
+    ipq2_filename = tmpdir.join("query2.txt").strpath
+
+    MarabouCore.saveQuery(ipq1, ipq1_filename)
+    MarabouCore.saveQuery(ipq1, ipq2_filename)
+    diff = call(['diff', ipq1_filename, ipq2_filename])
+    assert not diff
+
+
```

Comment:
Only one empty line should separate the assert and load_onnx_network

Reply:
Actually, even PEP-08 suggests surrounding top-level functions by two blank lines (and my editor, that I have now set to warn me on convention violations, complains when I try I remove this blank, so I've left it as is.

Reply:
I suggest adhering to the convention used in the other files in the repo